exporters/committime/collector_gitea.py

```diff
@@ -7,7 +7,7 @@
 
 from .collector_base import AbstractCommitCollector, UnsupportedGITProvider
 
-_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
+_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
The old code didn't have a Z. Something's funky here.

Reply:
Ah, the old code stripped away everything after the seconds with `str(timestamp)[:19]`, so it never saw the Z.

Are we sure it'll never give a fractional timestamp?